static/bidder-info/stroeerCore.yaml

```diff
@@ -0,0 +1,15 @@
+maintainer:
+  email: "help@cz.stroeer-labs.com"
+gvlVendorID: 136
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  iframe:
+    url: "https://js.adscale.de/pbsync.html?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&redirect={{.RedirectURL}}"
+  redirect:
+    url: "https://ih.adscale.de/uu?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&cburl={{.RedirectURL}}"
```

Comment:
Both user sync urls look good. If you're going to use uid concatenation instead of a macro, please specify an empty macro and add a comment, copying the style from `conntectad.yaml`.